sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableAddPartitionSuiteBase.scala

```diff
@@ -22,6 +22,20 @@ import org.apache.spark.sql.catalyst.analysis.PartitionsAlreadyExistException
 import org.apache.spark.sql.catalyst.catalog.CatalogTypes.TablePartitionSpec
 import org.apache.spark.sql.internal.SQLConf
 
+/**
+ * This base suite contains unified tests for the `ALTER TABLE .. ADD PARTITION` command that
+ * check V1 and V2 table catalogs. The tests that cannot run for all supported catalogs are
+ * located in more specific test suites:
+ *
+ *   - V2 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v2.AlterTableAddPartitionSuite]]
+ *   - V1 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuiteBase]]
```

Comment:
Unidoc doesn't work for the referencing in Javadoc with Scala's trait. it should be `` `org.apache.spark.sql.execution.command.v1.AlterTableAddPartitionSuite` ``

Reply:
Thank you very much @HyukjinKwon . Without your comment, I would spend much time in troubleshooting this.